src/include/sof/audio/pipeline.h

```diff
@@ -24,7 +24,7 @@
  * It should remain enabled in the situation when the
  * recovery is delegated to the outside of firmware.
  */
-#define NO_XRUN_RECOVERY 0
+#define NO_XRUN_RECOVERY 1
```

Comment:
This may be better as a Kconfig option.

Reply:
yes, as mentioned in the commit message, @slawblauciak will work on that, this commit is only for rescuing logs at Xrun happened, please refer to issue comment https://github.com/thesofproject/sof/issues/1578#issuecomment-506611258 for details.